templates/Asq.crontab

```diff
@@ -1,3 +1,3 @@
 SHELL=/bin/bash
 MAILTO=sys-msgs@yourdomain.com
-*/1 * * * * asq curl http://localhost/refresh >/dev/null 2>&1
+*/1 * * * * asq curl http://localhost:3000/refresh >/dev/null 2>&1
```

Comment:
Isn't this going to impact production environments where it's running on port 80?